picard/config.py

```diff
@@ -288,11 +270,18 @@ class FloatOption(Option):
 class ListOption(Option):
 
     convert = list
+    qtype = 'QVariantList'
+
+
+@staticmethod
+def _convert_to_int_list(values):
+    return list(map(int, values))
 
 
 class IntListOption(Option):
 
     convert = _convert_to_int_list
+    qtype = 'QVariantList'
```

Comment:
Just wanting to note that we don't seem to use `IntListOption` anywhere. I already thought about removing it in my previous PR, but was unsure whether we want to keep it around for plugins. Thoughts?

Reply:
I'd rather get rid of it, i'm not aware of any plugin using it.
And imho ListOption is enough